apps/console/src/features/identity-providers/components/settings/attribute-management/attribute-selection-wizard.tsx

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This file only has the license header changed. Please check and revert if unnecessary.

Reply:
Updated in b49a285